sdk/storage/azure-storage-blobs/test/ut/append_blob_client_test.cpp

```diff
@@ -338,4 +338,15 @@ namespace Azure { namespace Storage { namespace Test {
         appendBlobClient2.AppendBlockFromUri(appendBlobClient.GetUrl() + GetSas(), options2));
   }
 
+  TEST_F(AppendBlobClientTest, DISABLED_HighThroughputAppendBlob)
+  {
+    auto appendBlobClient = m_blobContainerClient->GetAppendBlobClient(RandomString());
+    appendBlobClient.Create();
+    auto blockContent = RandomBuffer(static_cast<size_t>(5_MB));
```

Comment:
Do not use large blobs in non-liveonly tests. This will explode the recoding file.

Reply:
Yeah, it's feature is not available now so I disable it. It will be a live only test. And the test record update is because Blob BlobServiceTest:SetProperties will expect a service veriosn:
```C++
    properties.DefaultServiceVersion = Blobs::_detail::ApiVersion;
    ...
    EXPECT_EQ(
        downloadedProperties.DefaultServiceVersion.HasValue(),
        properties.DefaultServiceVersion.HasValue());
```

Reply:
This is .net sdk's test: https://github.com/Azure/azure-sdk-for-net/pull/34548/files#diff-28f2ab7c9fb4e76c1f036da9a2fc840b078a11b753a0d66fc943c5bc148ce25fR1142